### PR TITLE
heroku 9.5.1 (new formula)

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Removed in https://github.com/Homebrew/homebrew-core/pull/33233, back by @p-linnane demand

### CODE REVIEW

**Review by p-linnane — APPROVED** [review]

**Review by iMichka — CHANGES_REQUESTED** [review]
We removed it here: https://github.com/Homebrew/homebrew-core/pull/33233 ?

File: Formula/h/heroku.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      terminal_notifier_dir = node_notifier_vendor_dir/"mac.noindex"
+      terminal_notifier_dir.mkpath
+
+      # replace vendored `terminal-notifier` with our own
+      terminal_notifier_app = Formula["terminal-notifier"].opt_prefix/"terminal-notifier.app"
+      ln_sf terminal_notifier_app.relative_path_from(terminal_notifier_dir), terminal_notifier_dir
+    end
+
+    # Replace universal binaries with their native slices.
+    deuniversalize_machos
```
Comment (chenrui333) [inline Formula/h/heroku.rb:30]: we probably dont need this?

```suggestion
```

Comment (SMillerDev) [reply Formula/h/heroku.rb:30]: Yeah we do, there's a universal binary in there

**Review by bevanjkay — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Review by iMichka — APPROVED** [review]
We discussed this internally: it's ok and we can ship it

File: Formula/h/heroku.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  bottle do
+    sha256 arm64_sequoia: "c6118421b58f3f14972958fe4fff97d486e633899aede9cd4abfd7a35d7a9a5f"
+    sha256 arm64_sonoma:  "e5e891e569dbb21b3e28b2aefc24700eed95ac97dfb4d49db162467c42ff2308"
+    sha256 arm64_ventura: "c0322f0b6c79f0c72941d7e1a59ab716136298abdd5f003e0cdf17fff6350d4a"
+    sha256 sonoma:        "bc923961ae796a43b7f2adf8885c946f8419a0628c286c137dc39cb62ebac1c0"
+    sha256 ventura:       "0edccb3453122d52dbd33a30bf052d3d5936e7db01dda63de13fea5a5f7abd36"
+  end
+
+  depends_on "node"
+  depends_on "terminal-notifier"
```
Comment (chenrui333) [inline Formula/h/heroku.rb:17]: actually this should be macos only dep. I will followup on this :)

**Pull request merged**